accounts-db/src/accounts_db.rs

```diff
@@ -4341,43 +4367,14 @@ impl AccountsDb {
                 // clean needs to take care of this dead slot
                 self.accounts_index.add_uncleaned_roots([slot]);
             }
-            info!(
```

Comment:
this should remain. the ohter pr just puts an `if` around it to quiet the noise.

Reply:
yeah. done.